presto-spi/src/main/java/com/facebook/presto/spi/type/NamedTypeSignature.java

```diff
@@ -26,19 +26,31 @@
 {
     private final Optional<String> name;
     private final TypeSignature typeSignature;
+    private final boolean quoted;
```

Comment:
The proper name is "delimited" (see Identifier). Also, both the field name and the type could be quoted.